Unified/go_condor.py

```diff
@@ -11,6 +11,41 @@
 from collections import defaultdict
 
 
+def makeHoldAds(config):
+    """
+    Create a set of rules to hold a task from matching
+    """
+    for task,where in config.get('hold',{}).items():
+        # task is the task name
+        # where is either an empty list=all sites, or a list of sites (not implemented)
+        anAd = classad.ClassAd()
+        anAd["Name"] = str("Holding task %s from %s"%(task, where))
+        anAd["GridResource"] = "condor localhost localhost"
+        anAd["TargetUniverse"] = 5
```

Comment:
Is it going to set the condor JobStatus to 5 (Held)? If so, beware held jobs are considered an error status in WMCore